unstract/connectors/src/unstract/connectors/filesystems/minio/static/json_schema.json

```diff
@@ -32,12 +32,6 @@
       "title": "Bucket Name",
       "description": "Name of the bucket to be restricted to."
     },
```

Comment:
QQ : If we remove the path from the schema, will we not support adding files in the nested folders?

Reply:
yes, tested this